apps/web-mzima-client/src/app/shared/components/collections/collections.component.ts

```diff
@@ -14,6 +14,8 @@ import {
   PostResult,
   AccountNotificationsInterface,
 } from '@mzima-client/sdk';
+// eslint-disable-next-line @nx/enforce-module-boundaries
+import { AlertService } from 'libs/sdk/src/lib/services/alerts.service';
 import { BaseComponent } from '../../../base.component';
```

Comment:
You don't need to add a new alert-service, we already have a service called notification-service you can use, look for how that is used in the rest of the codebase: https://github.com/ushahidi/platform-client-mzima/blob/development/apps/web-mzima-client/src/app/core/services/notification.service.ts